src/self-model.scm

```diff
@@ -583,6 +583,7 @@
 (DefineLink
 	(DefinedPredicate "look at person")
 	(SequentialAnd
+		(DefinedPredicate "Time to change gaze")
```

Comment:
Me and Mandeep were working on this together. The way we went at is to make minimal change as possible without unraveling the btree. It is a bit hard following through the btree as it is intertwined. And we didn't test with chatbot-eva though.

You are right that each functionality should be part of independent psi-demand. That is why I have separated out tracking from face-demand. Once this demo is over,  I will break the btree to atomic demands with two overall stages
`update-internal-and-external-modle-demands -> orcheterate-behavior-demands.`.